src/pages/welcome/stepOne/index.js

```diff
@@ -1,8 +1,32 @@
 import ProfileIcon from "../../../assets/icons/profileIcon"
 import Form from "../../../components/form"
 import TextInput from "../../../components/form/textInput"
+import {useState} from "react";
 
 const StepOne = ({ data, setData }) => {
+    const [profilePicture, setProfilePicture] = useState("")
```

Comment:
maybe add Base64 at the end of the variable name, so we know it's a base 64, since in react we could import images